### DISCUSSION TITLE
How to Unban account

### DISCUSSION THREAD

**Discussion by sunscreen** [rustdesk/rustdesk#8197] [General]
Hi I got locked out of a client, too many wrong password attempts .. how to unban?

**Comment by rustdesk** [comment]
Reboot controlled machine.

**Reply by sunscreen** [reply]
Thanks 👍

**Reply by sunscreen** [reply]
Would restarting the rustdesk service also surfice?

**Reply by sunscreen** [reply]
If any one else neesd to know .. yes restarting the service rather then restarting the entire client is a option
